docs/dev_install.rst

```diff
@@ -75,9 +75,12 @@ Setting up Taichi for development
     mkdir build
     cd build
     cmake ..
+    # if you donot set clang as the default compiler in Linux
```

Comment:
nit:
```suggestion
    # if you do not set clang as the default compiler
```
1. Please use `do not` for clarity.
2. Adding `clang` to `PATH` can be also applied to OS X and Windows.

Reply:
Thanks! please give me a while to learn your 2) suggestion then I will commit these suggestions.